pages/main/script.js

```diff
@@ -0,0 +1,112 @@
+function Builder(value) {
+        this.isInteger = Number.isInteger(value);
+        this.curentValue = (this.isInteger) ? value : String(value);
+};
+
+Builder.prototype.get = function() {
+    return this.curentValue;
+};
+
+Builder.prototype.plus = function(...args) {
+    for (let item of args) {
+        this.curentValue += item;
+    }
+    return this;
+}
+
+Builder.prototype.minus = function(...args) {
+    if (this.isInteger) {
+        for (let item of args) {
```

Comment:
FYI. Because you transformed arguments to the array with help of the spread operator, you can also use forEach.